test/functional/rpc_dumptxoutset.py

```diff
@@ -42,16 +39,38 @@ def run_test(self):
         with open(str(expected_path), 'rb') as f:
             digest = hashlib.sha256(f.read()).hexdigest()
             # UTXO snapshot hash should be deterministic based on mocked time.
-            assert_equal(
-                digest, '7ae82c986fa5445678d2a21453bb1c86d39e47af13da137640c2b1cf8093691c')
+            assert_equal(digest, expected_digest)
 
         assert_equal(
             out['txoutset_hash'], 'd4b614f476b99a6e569973bf1c0120d88b1a168076f8ce25691fb41dd1cef149')
         assert_equal(out['nchaintx'], 101)
 
         # Specifying a path to an existing file will fail.
         assert_raises_rpc_error(
-            -8, '{} already exists'.format(FILENAME),  node.dumptxoutset, FILENAME)
+            -8, '{} already exists'.format(filename),  node.dumptxoutset, filename)
+
+        if (is_human_readable):
+            with open(expected_path, 'r', encoding='utf-8') as f:
+                content = f.readlines()
+                assert_equal(content[0].rstrip(),
+                    "#(blockhash 6fd417acba2a8738b06fee43330c50d58e6a725046c3d843c8dd7e51d46d1ed6),txid,vout,value,coinbase,height,scriptPubKey")
+                assert_equal(content[1].rstrip(),
+                    "213ecbdfe837a2c8ffc0812da62d4de94efce8894c67e22ff658517ecf104e03,0,5000000000,1,81,76a9142b4569203694fc997e13f2c0a1383b9e16c77a0d88ac")
+
+    def run_test(self):
+        """Test a trivial usage of the dumptxoutset RPC command."""
+        node = self.nodes[0]
+        mocktime = node.getblockheader(node.getblockhash(0))['time'] + 1
+        node.setmocktime(mocktime)
+        self.generate(node, COINBASE_MATURITY)
+
+        txt_file_digest = '5bc8a9c14d1f6d89833342dcd6014bdf9ddb5f19e3741760da6d6d666971df41'
+
+        if os.name == 'nt':
+            txt_file_digest = 'dff298e77a80bb6e292526c0729d78fdef68a537c372563b5f5dbfadddabc780'
```

Comment:
This is kind of an ugly hack. Maybe we just need to open the file in text mode and hash it as if it were UNIX format? :/

Reply:
Python's text mode reader is overly tolerant, so wouldn't check the EOL used is correct.

Instead, I ended up with https://github.com/luke-jr/bitcoin/commit/58e0d32de6a444bd73d30355ffe17dc7ed96e738 - but it may be better off just leaving this alone for this PR (unless we care about the added flexibility of supporting any arbitrary platform)

Reply:
Done. Thanks for the detailed review.